### ISSUE TITLE
Support for overriding table row colors?

### ISSUE THREAD

**Issue by kimsauce** [rohit-gohri/redocusaurus#199]
Hello Rohit, thank you for making this great plugin! I was wondering if there's a way to change the table row colors. I'm currently seeing an issue where they're rendering like this in dark mode. I've tried several ways, but now stumped. 

<img width="700" alt="darkmode-rows" src="https://user-images.githubusercontent.com/56411016/172984136-3d9cabc3-2981-4e50-9ff5-55824da7ffe9.png">

We're also seeing a code block color contrast issue, in light mode only: 
<img width="502" alt="2022-08-24_09-59-08" src="https://user-images.githubusercontent.com/56411016/186478590-27458a95-184d-4642-beb9-03cdb9ea6772.png">

**Comment by rohit-gohri** [comment]
Can you share an example OpenAPI/Swagger yaml that recreates this?

**Comment by kimsauce** [comment]
Sure thing! Here you go, and many thanks for the quick response.

[sumologic-api.yaml.zip](https://github.com/rohit-gohri/redocusaurus/files/8876168/sumologic-api.yaml.zip)

**Comment by kimsauce** [comment]
Hi @rohit-gohri , I'm still working on this one and feel like I'm getting closer! It may be something on our end, but could I run these ideas past you? 

Our issues are the table row poor color contrast (in dark mode) and code snippet poor color contrast (in light mode). For your reference, here's our [redocusaurus page](https://d10g313yy2pc88.cloudfront.net/sumoapi) and [Redocly page](https://api.sumologic.com/docs/) . 

- Is it possible to import CSS directly into a yaml spec file?
- Does redocusaurus accept override CSS or entire CSS files? 
<img width="654" alt="redoc" src="https://user-images.githubusercontent.com/56411016/186477278-716fa5ca-b45a-471b-a165-fc1fd27752e1.png">
<img width="726" alt="redoc2" src="https://user-images.githubusercontent.com/56411016/186477310-70494643-c46f-4b9b-bea3-f18c463cf16a.png">

**Comment by rohit-gohri** [comment]
Hey, thank you for taking the time to get to the bottom 

If you have fixed this with some custom css then I would be happy to merge the changes into the package itself (as long as it's not specific to your spec file). You can add your changes here and open a PR: https://github.com/rohit-gohri/redocusaurus/blob/main/packages/docusaurus-theme-redoc/src/theme/Redoc/styles.css 

As for your questions:

> Is it possible to import CSS directly into a yaml spec file?
> Does redocusaurus accept override CSS code or entire CSS files in docusaurus.config.js?

The answer would be no to both, but redocusaurus is just React components and some config. So you could ideally just import the React component and use it directly like in this example: https://github.com/rohit-gohri/redocusaurus/blob/main/website/src/pages/examples/custom-page/index.jsx . And use import the custom css in that file.

**Comment by kimsauce** [comment]
Hi @rohit-gohri , I came back to this (almost a year later) and was able to resolve the issue using a custom class in our CSS!

```css
.redocusaurus table > tbody > tr {
  border: 1px solid #F1F1F1;
  background-color: transparent;
  color: #000000;
}
```

Do you think this would work across the board for all users, they want to override? I've just picked plain colors here, but other users can choose what they want.

**Comment by rohit-gohri** [comment]
That looks good to me! Thanks for posting it back

> I've just picked plain colors here

Maybe we can use css variables from the infima theme that docusaurus uses - https://github.com/facebookincubator/infima/blob/main/packages/core/styles/common/variables.pcss rather than hardcoded values?

**Comment by kimsauce** [comment]
Oh good call. This worked for me! 

```
.redocusaurus table > tbody > tr {
  background-color: var(--ifm-table-cell-color) !important;
}
```

**Comment by kimsauce** [comment]
Sure thing! Just created a PR for it [here](https://github.com/rohit-gohri/redocusaurus/pull/295). I wasn't sure whether to put the code there and/or in the [Docs](https://redocusaurus.vercel.app/docs/getting-started/theme-options).

Thanks again for your help! Excited to use Redocusaurus 🦖

**Comment by rohit-gohri** [comment]
Released in 1.6.3
